ethereum/beaconchain/src/main/java/tech/pegasys/artemis/state/StateTransition.java

```diff
@@ -38,9 +37,13 @@ public void initiate(BeaconState state, BeaconBlock block) {
     }
 
     // per-epoch processing
-    if (state.getSlot() % Constants.EPOCH_LENGTH == 0) {
+    // if (state.getSlot() % Constants.EPOCH_LENGTH == 0) {
```

Comment:
You should leave this uncommented.  Code that other people are working on need the epoch processing to only happen every 64 slots.